magento/app/design/frontend/base/default/layout/page.xml

```diff
@@ -48,6 +48,8 @@ Default layout, loads most of the pages
                 <action method="addJs"><script>varien/menu.js</script></action>
                 <action method="addJs"><script>mage/translate.js</script></action>
                 <action method="addJs"><script>mage/cookies.js</script></action>
+                <action method="addJs"><script>footer_js/jquery.min.js</script></action>
+                <action method="addJs"><script>footer_js/footer.js</script></action>
```

Comment:
instead of making footer.js make theme.js which will be applicable for entire theme